src/lowparse/LowParse.Spec.BitSum.fst

```diff
@@ -18,7 +18,7 @@ type bitsum'
     bitsum' cl bitsum'_size
 | BitSum' :
     (key: eqtype) ->
-    (key_size: nat { key_size > 0 /\ key_size <= bitsum'_size /\ bitsum'_size <= tot }) -> // key_size made positive because F* cannot prove that (payload _) is a smaller term wrt. << without FStar.WellFounded.axiom1_dep
+    (key_size: nat { key_size > 0 /\ key_size <= bitsum'_size /\ bitsum'_size <= tot }) -> // key_size made positive because F* cannot prove that (payload _) is a smaller term wrt. << without FStar.WellFounded.axiom1_dep //NS: TODO: please check this comment
```

Comment:
Thank you Nik for pointing this out. For now I will leave it as is, for the sake of merging your changes into master, CI permitting.
Later I will separately investigate whether I can remove that restriction on `key_size`, and if not, I will update the comment accordingly.